Added wooden low bridges

Mailaender opened the pull request:
Closes https://github.com/OpenRA/ra2/issues/329. Depends upon https://github.com/OpenRA/OpenRA/pull/13882.

Also brings back Arctic Circle. It was trying to import `caoutp` so I added that as well in placeholder quality as that wasn't the intent of this, but I still didn't want it missing in that map.

LinuxDonald commented:
Can this not get merged yet?

abcdefg30 commented:
It needs to be rebased first.

Mailaender commented:
```diff
<<<<<<< low-bridges
ENGINE_VERSION="d5f63f256a74263166bf442394c2e0661c9457e4"
=======
ENGINE_VERSION="2a3a27"
>>>>>>> master
```

I have no idea what is the right version. Those hashes are just cryptic to me.

Mailaender commented:
Can https://github.com/OpenRA/ra2/pull/393/commits/45594e5cb0dbf99accdabbe697f6ec3e73c41078 be removed now?

LinuxDonald commented:
Maybe Engine version is git commit hash?

Mailaender commented:
Rebased though I temporarily had to re-add https://github.com/OpenRA/OpenRA/pull/13802 but exclude the attack move.

LinuxDonald commented:
Anyone up for review this? :(

Mailaender commented:
There is a problem with the commandline checks that I don't understand:

```
OpenRA.Utility(1,1): Error: Failed with exception: System.IO.FileNotFoundException: File not found: ^maps/ra2/{DEV_VERSION}
File name: '^maps/ra2/{DEV_VERSION}'
  at OpenRA.FileSystem.FileSystem.Open (System.String filename) [0x00031] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Game/FileSystem/FileSystem.cs:216 
  at OpenRA.FileSystem.FileSystem.OpenPackage (System.String filename) [0x00051] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Game/FileSystem/FileSystem.cs:74 
  at OpenRA.Mods.Common.UtilityCommands.CheckYaml.OpenRA.IUtilityCommand.Run (OpenRA.Utility utility, System.String[] args) [0x001f3] in /home/travis/build/OpenRA/ra2/engine/OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs:93 
make: *** [test] Error 1
```

DylanC commented:
@abcdefg30 - Can @Mailaender get help with this error? (please)

abcdefg30 commented:
Sorry, I don't know when I find the time to focus on RA2, especially as I'm occupied with "real life" issues at least until the end of February.

DylanC commented:
@abcdefg30 - No problem. Thanks for the reply.

Mailaender commented:
@abcdefg30 silently added the tech outpost in https://github.com/OpenRA/ra2/commit/0490d781827b92c94cbb9661ffcfe2bf293e6a8f so I removed my placeholder here. Thanks to @MustaphaTR we have a command bar https://github.com/OpenRA/ra2/pull/451 and I could safely remove the UnitCommandWidget.

I now updated my branch to the next playtest. This does not only fix the bridge SHPs, but also the crate effect animations so I enabled those as well.

Mailaender commented:
I only moved this a few commits forward now as I faced a wall of actor rules that need manual corrections after the upgrade.

Mailaender commented:
Also managed to find the fix for https://github.com/OpenRA/OpenRA/issues/14936 on my own.

MustaphaTR commented on mods/ra2/sequences/misc.yaml:
```diff
@@ -56,19 +56,17 @@ beacon:
 		Length: 1
 		ZOffset: 2047
 
-crate-effects: #Temp Null Some SHPs cannot be read by openra at this time, expect more
```
You deleted the whole line instead of just the comment.

Mailaender replied:
Fixed.

MustaphaTR commented:
I'm not sure if it is the same thing @abcdefg30 is compaining on update engine to lastest release PR, but here too, sounds are broken. Everthing that is in audio.bag plays (i think) the chorno leginnerrie fire sound.

abcdefg30 commented:
I already took enough of your time with this, so I just went ahead and did the last fixes myself:
- Removed some redundant values from the new map (like `Health: 100`)
- Allow huts to connect from two cells distance (like in the original) which makes all bridges work again

Sorry again for the delay.

Mailaender commented:
Thanks for the assistance. I was going a bit mad with the bridge hut connectivity.
